sdk/storage/inc/blobs/blob_client.hpp

```diff
@@ -284,6 +284,76 @@ namespace Azure { namespace Storage { namespace Blobs {
     Azure::Core::Response<UndeleteBlobInfo> Undelete(
         const UndeleteBlobOptions& options = UndeleteBlobOptions()) const;
 
+    /**
+     * @brief Acquires a lease on the blob.
+     *
+     * @param proposedLeaseId
+     * Proposed lease ID, in a GUID string format.
```

Comment:
The format of the comment here is really weird.

Reply:
It is weird

Reply:
Can you try to fix it?

Reply:
not a priority